first version of the German localization file

VinceMasuka opened the pull request:
Please keep in mind modify the localization section in user_config.h to "#define MY_LANGUAGE            de-DE".

VinceMasuka committed:
first version of German localization file

Please keep in mind modify the localization section in user_config.h to "#define MY_LANGUAGE            de-DE"

arendst commented:
Due to excessive string lenght I needed to adjust some source fields. Fixed. Will be released in next (pre)release.

VinceMasuka commented:
Awesome! Thanks for the info! 
-- 
Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 16.10.17, 11:40, Theo Arends <notifications@github.com> schrieb:
Due to excessive string lenght I needed to adjust some source fields. Fixed. Will be released in next (pre)release.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/arendst/Sonoff-Tasmota","title":"arendst/Sonoff-Tasmota","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/arendst/Sonoff-Tasmota"}},"updates":{"snippets":[{"icon":"PERSON","message":"@arendst in #1022: Due to excessive string lenght I needed to adjust some source fields. Fixed. Will be released in next (pre)release."}],"action":{"name":"View Pull Request","url":"https://github.com/arendst/Sonoff-Tasmota/pull/1022#issuecomment-336834770"}}}

arendst commented:
In pre-release 5.8.0m
